apps/client/src/features/rundown/event-editor/composite/EventEditorTitles.tsx

```diff
@@ -46,7 +48,10 @@ const EventEditorTitles = (props: EventEditorTitlesProps) => {
       </div>
       <div>
         <Editor.Label>Colour</Editor.Label>
-        <SwatchSelect name='colour' value={colour} handleChange={handleSubmit} />
+        <div className={style.colorSettings}>
```

Comment:
unfortunately this still does not work for cases where the list must wrap 

I wonder if the easiest thing is to include `EventColorPicker` in the `SwatchSelect` list?

<img width="471" alt="Screenshot 2024-12-21 at 10 12 10" src="https://github.com/user-attachments/assets/fe18c534-56c4-4891-b0c4-f21a45d4cd3d" />